docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc

```diff
@@ -1,18 +1,40 @@
 
 = BGP Monitoring Protocol
+:description: Learn how the BGP Monitoring Protocol (BMP) integration in OpenNMS Horizon/Meridian helps you to see what the Border Gateway Protocol is doing.
 
-The BGP Monitoring Protocol (BMP) provides a convenient interface for obtaining route views.
-The integration in {page-component-title} lets you use these route views, status updates, and statistics for monitoring and management.
+The https://datatracker.ietf.org/doc/html/rfc7854[BGP Monitoring Protocol] (BMP) monitors Border Gateway Protocol (BGP) sessions and provides a convenient interface to monitor BGP routing information on the routing device.
+
+The BMP integration in {page-component-title} lets you use these route views, status updates, and statistics for advanced monitoring and management, providing visibility into what BGP is doing.
+
+== OpenNMS BMP integration
+
+OpenNMS leverages https://www.snas.io/[OpenBMP], an application that was developed as a reference implementation for receiving BMP messages from network equipment and presenting the state of the BGP network to the user.
+
+BGP runs on routers and high-end switches.
+Those devices can point to {page-component-title} or an OpenNMS Minion and send BMP data to {page-component-title}.
```

Comment:
"point to opennms horizon or an opennms minion and send bmp data to opennms horizon" feels a little awkward? not sure how to change it tho

Reply:
Good catch. Thanks. I'll see what I can do. No worries about the placeholder questions. Perhaps Mark knows.